website/docs/r/virtual_machine.html.markdown

```diff
@@ -276,6 +276,23 @@ For more information on the different example configurations, please check out t
 * `image_uri` - (Optional) Specifies the image_uri in the form publisherName:offer:skus:version. `image_uri` can also specify the [VHD uri](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-linux-cli-deploy-templates/#create-a-custom-vm-image) of a custom VM image to clone. When cloning a custom disk image the `os_type` documented below becomes required.
 * `os_type` - (Optional) Specifies the operating system Type, valid values are windows, linux.
 * `disk_size_gb` - (Optional) Specifies the size of the os disk in gigabytes.
+* `encryption_settings` - (Optional) Species the encryption settings for the OS disk as documented below. 
+See [Azure Disk Encryption for Windows and Linux IaaS vms](https://docs.microsoft.com/en-us/azure/security/azure-security-disk-encryption) 
+for more information.
+
+`encryption_settings` supports the following:
+
+* `enabled` - (Required) Specifies if the encryption is enabled. If false, disk_encryption_key and key_encryption_key must not be specified.
+* `disk_encryption_key` - (Optional) Specifies the location of the disk encryption key using `secret_url` and `source_vault_id` subproperties. Must be specified if enabled is true.
```

Comment:
Given this is a block not a property - we'd tend to phrase this as:
```
`disk_encryption_key` - (Optional) A `disk_encryption_key` block as defined below. Required if `enabled`s is set to `true`.
```

Could we also define the properties in the block below?

Reply:
done